pkg/virt-launcher/virtwrap/network/network.go

```diff
@@ -51,8 +51,8 @@ func SetupNetworkInterfaces(vmi *v1.VirtualMachineInstance, domain *api.Domain)
 	cniNetworks := map[string]int{}
 	for _, network := range vmi.Spec.Networks {
 		networks[network.Name] = network.DeepCopy()
-		if networks[network.Name].Multus != nil {
-			// multus pod interfaces start from 1
+		if networks[network.Name].Npwg != nil || networks[network.Name].Multus != nil {
```

Comment:
```networks[network.Name].IsMultusOrNpwg()``` same with the checks similar checks later on in this function.

Reply:
done